.github/workflows/release-notes-issue.yml

```diff
@@ -0,0 +1,106 @@
+# release-notes-issue.yml
+# Updates a release notes tracking issue with the generated release notes.
+# The issue to update is set in the RELEASE_NOTES_ISSUE secret.
+
+name: "Update release notes issue"
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - ".changeset/**"
+
+  # Allow manually triggering this workflow from the web UI
+  workflow_dispatch:
+    inputs:
+      issueNumber:
+        description: 'Issue number to update'
+        default: 22101
+        required: true
+
+permissions:
+  issues: write
+
+jobs:
+  create-issue:
```

Comment:
For anyone else who made need to make changes to this in the future, I think more in-depth step-by-step documentation explaining what's going on would be helpful. Especially since most devs on our team don't work with these pipeline templates often.

Reply:
I added a bunch of comments.

Reply:
I added a bunch of comments.